index.html

```diff
@@ -1,10 +1,521 @@
-<<!DOCTYPE html>
-<html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <title></title>
-  </head>
-  <body>
-
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>NY Times</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="/vi-assets/static-assets/favicon-4bf96cb6a1093748bf5b3c429accb9b4.ico">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/main.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <i class="material-icons">menu</i>
+            <i class="material-icons">search</i>
+            <span>SPACE & COSMOS</span>
+            <img src="./image/NYT-Logo.png" alt="logo" height="25">
+            <button>SUBSCRIBE NOW</button>
+            <button>LOG IN</button>
+        </nav>
+    </header>
+
+    <hr>
+    <section>
+        <!-- section div start-->
+        <span id="headline">Space Ripples Reveal <br> Big Bang’s Smoking Gun</span>
+        <div>
+            <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp"
+                alt="" width="900">
+            <p class="img1">
+                Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains
+                how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8
+                billion
+                years ago.CreditCreditRick Friedman for The New York Times</p>
+        </div>
+        <span style="font-weight: 700">By Dennis Overbye</span>
+        <div>
+            <p style="text-align: center">March 17, 2014</p>
```

Comment:
The author and the date are not aligned to the center in the original article. Also, watch the spaces (And don't use `br` for spacing. Use CSS)